src/queries/proposals.ts

```diff
@@ -138,3 +138,20 @@ export const getProposalById = (id: string): Promise<any> => {
     `;
   return processRequest(query);
 };
+
+export const getProposalsFromPlatformServices = (id: string): Promise<any> => {
+  const query = `
+      {
+          proposals(where: {service_: {platform: "${id}"}}) {
```

Comment:
need to be ordered to have last one first. Actually without it the cron will stop working when he will reach the default limit

Reply:
We would probably need more info to construct the email: handle, rate amount...